itkwidgets/widget_viewer.py

```diff
@@ -624,6 +628,67 @@ def view(image=None,
         widget.
     """
 
+    # this block allows the user to pass already formed vtkActor vtkVolume objects
+    actors = kwargs.pop("actors", None)
+    if actors is not None:
+        if not isinstance(actors, (list, tuple)): # passing the object directly, so make it a list
+            actors = [actors]
+
+        geometries, point_sets, images = [],[],[]
```

Comment:
To make it possible to also pass `geometries`, `point_sets`, and `images`, can we avoid re-initializing as empty arrays here, i.e. take advantage of the default arguments. Te enabling use of the default values for the colors and opacities, we should insert into the front of the color and opacities lists instead of appending.

Reply:
i\m not sure why we should inserting instead of appending, but I made a couple of tests and it seems to work by just removing the initialization as you suggested!